test/e2e/testdata/ext-auth-http-service.yaml

```diff
@@ -66,6 +66,12 @@ spec:
         volumeMounts:
         - name: http-ext-auth
           mountPath: /usr/src/app
+        readinessProbe:
```

Comment:
curious why this is needed ?

Reply:
Because I want to avoid the test calls the ext auth service before it's ready. In that case, the test would fail.